pkg/appconsts/v2/app_consts.go

```diff
@@ -4,7 +4,4 @@ const (
 	Version              uint64 = 2
 	SquareSizeUpperBound int    = 128
 	SubtreeRootThreshold int    = 64
-	// NetworkMinGasPrice is used by x/minfee to prevent transactions from being
-	// included in a block if they specify a gas price lower than this.
-	NetworkMinGasPrice float64 = 0.000001 // utia
```

Comment:
[nit] another breaking change we could add to changelog / v3 release notes.

Reply:
where are the release notes?

Reply:
we don't have a place to store it yet so no change needed. See [Slack thread](https://celestia-team.slack.com/archives/C02PFFCGMNW/p1723141674451709) for more context

Reply:
We now have a place for this: https://github.com/celestiaorg/celestia-app/blob/main/docs/release-notes/pending-release.md